docs/arlas-collection-model.md

```diff
@@ -80,6 +80,9 @@ The most important fields are:
     ```
     `labels.status` being the taggable field.
 
+!!! info "Important 3"
+    `geometry_path` field value must have the same format in all documents: It's not recommended to index documents where `geometry_path` value is WKT and others as GeoJson. Otherwise a ParseException will be thrown. It goes the same for `centroid_path`.
```

Comment:
More than "not recommended", it's "not allowed" to store geometries in `geometry_path` and/or `centroid_path` as WKT and GeoJson at the same time.